features/docs/gegeven-stap-definities-expressief.feature

```diff
@@ -74,11 +74,11 @@
       | stap | categorie    | text                                                                                                                                                                                                         | values                                      |
       | 1    | inschrijving | INSERT INTO public.lo3_pl(pl_id,mutatie_dt,geheim_ind) VALUES((SELECT COALESCE(MAX(pl_id), 0)+1 FROM public.lo3_pl),current_timestamp,$1) RETURNING *                                                        | 0                                           |
       |      | persoon      | INSERT INTO public.lo3_pl_persoon(pl_id,stapel_nr,volg_nr,persoon_type,burger_service_nr,geslachts_naam) VALUES($1,$2,$3,$4,$5,$6)                                                                           | 9999,0,0,P,000000012,P2                     |
-      |      | partner      | INSERT INTO public.lo3_pl_persoon(pl_id,volg_nr,burger_service_nr,geslachts_naam,relatie_start_datum,relatie_start_plaats,relatie_start_land_code,persoon_type,stapel_nr) VALUES($1,$2,$3,$4,$5,$6,$7,$8,$9) | 9999,0,000000024,P3,20220925,0518,6030,R,0  |
+      |      | partner      | INSERT INTO public.lo3_pl_persoon(pl_id,volg_nr,burger_service_nr,geslachts_naam,relatie_start_datum,relatie_start_plaats,relatie_start_land_code,persoon_type,stapel_nr) VALUES($1,$2,$3,$4,$5,$6,$7,$8,$9) | 9999,0,000000024,P3,gisteren - 2 jaar,0518,6030,R,0  |
```

Comment:
voor de standaard situatie zou ik het logischer vinden dat de relatie start voor de geboorte van het kind. Aangezien "is minderjarig" een geboortedatum maakt met gisteren - 17 jaar, zou de startdatum van de relatie van ouders m.i. daarvoor moeten liggen. Bijvoorbeeld gisteren - 20 jaar.

Reply:
startdatum van de relatie van ouders is aangepast naar gisteren - 20 jaar